### PR TITLE
Add homepage for choosing a station

### PR DESCRIPTION
**Pull request by muyjohno**
Also add some links to the menu

image
image

### CODE REVIEW

File: app/stylesheets/components/_header.scss [hunk 1]
```diff
@@ -23,11 +23,15 @@
 
 .menu {
   background: $color-black;
-  padding: 20px;
+  padding: 10px;
   position: absolute;
   right: 0;
   top: $header-height;
 
+  li {
```
Comment (keefyhub) [inline app/stylesheets/components/_header.scss]: can we put this on the `&__item`, we are trying not to target elements unless its a global change

File: app/stylesheets/components/_room-selector.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  text-align: center;
+  width: 100%;
+}
+
+.room-selector__field {
+  @include font-size(9)
+  background-color: $color-black;
+  border: 2px solid $color-light-gold;
+  border-radius: 50px;
+  clear: left;
```
Comment (keefyhub) [inline app/stylesheets/components/_room-selector.scss]: does this clear do anything?

File: app/stylesheets/components/_room-selector.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  color: $color-white;
+  display: block;
+  font-weight: 700;
+  height:100px;
```
Comment (katiewelton) [inline app/stylesheets/components/_room-selector.scss]: being pedantic - can you add a space between the : and the 100px?

**Comment by katiewelton** [comment]
👍  after the 2 comments

**Pull request merged**